src/return.c

```diff
@@ -112,7 +112,9 @@ void uopz_get_return(zend_class_entry *clazz, zend_string *function, zval *retur
 		return;
 	}
 
-	ureturn = zend_hash_find_ptr(returns, function);
+	zend_string *key = zend_string_tolower(function);
```

Comment:
For consistency with other functions in that file (and likely the rest of the code base), please move the declaration of `key` to the beginning of the function.

Reply:
Sure. Will be updated shortly.

Reply:
Done :+1: